src/Radar.js

```diff
@@ -0,0 +1,237 @@
+/**
+    @external Viz
+    @see https://github.com/d3plus/d3plus-viz#Viz
+*/
+import {nest} from "d3-collection";
+import * as d3 from "d3-selection";
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Circle, Path} from "d3plus-shape";
+import {TextBox} from "d3plus-text";
+
+import {default as Plot} from "./Plot";
+
+const tau = Math.PI * 2;
+
+/**
+    @class Radar
+    @extends Plot
+    @desc Creates a radar visualization based on an array of data.
+*/
+export default class Radar extends Plot {
+
+  /**
+      @memberof LinePlot
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from Plot.
+      @private
+  */
+  constructor() {
+    super();
+    this._discrete = "x";
+    this._levels = 6;
+
+    this._on.mouseenter = () => {};
+    this._on["mouseleave.shape"] = () => {
+      this.hover(false);
+    };
+    const defaultMouseMove = this._on["mousemove.shape"];
+    this._on["mousemove.shape"] = (d, i) => {
+      defaultMouseMove(d, i);
+      const id = this._id(d, i);
+
+      this.hover((h, x) => this._x(h, x) === id);
+    };
```

Comment:
these mouseevents do not do anything different from the default events, please remove.